docs/frontend-api/introduction-to-frontend-api.md

```diff
@@ -61,6 +61,27 @@ And you get back following JSON containing names of all root categories on the d
 Frontend API respects the domain you call so in case of the standard two domain setup with default data fixtures,
 you can get data for the first domain with the request made to `http://127.0.0.1:8000/graphql/` while requesting `http://127.0.0.2:8000/graphql/` returns data for the second domain.
 
+###  Requesting API from another domain (handling CORS)
+[overblog/GraphQLBundle](https://github.com/overblog/GraphQLBundle) comes out of the box with a generic and simple [CORS (Cross-Origin Resource Sharing)](http://enable-cors.org/) handler.
+The handler is disabled by default, so your API is better protected from attacks by malicious scripts.
+
+You can enable the simple CORS handler in `config/packages/shopsys_frontend_api.yaml` file:
+
+``` diff
+    overblog_graphql:
++       security:
++           handle_cors: true
+        definitions:
+            schema:
+            # ...
+```
+
+You can read more in [OverblogGraphQLBundle documentation](https://github.com/overblog/GraphQLBundle/blob/v0.13.4/docs/security/handle-cors.md)
+
+!!! note
```

Comment:
```suggestion
!!! warning
```

I would not use just note here. This is really important information to me. When I am doing such a change it will be completely opened to the world and I should be really well informed about it.